static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -62,8 +63,7 @@ const ErrorInfoChecks: Array<ErrorInfo> = [
     errorHelpType: ErrorHelpType.DYNAMIC_SERVER_USAGE,
   },
   {
-    errorTitle:
-      /(does not match server-rendered HTML|Hydration failed because|error while hydrating)/i,
+    errorTitle: isHydrationError,
```

Comment:
I refactored this out so i can re-use it inside the issue-details page to show a diff of the page content, if a replay is attached. 

A replay could be attached to errors of issueType `IssueType.ERROR` and will be attached if the issueType is `IssueType.REPLAY_HYDRATION_ERROR`